arch/platform/zoul/dev/grove-gyro.c

```diff
@@ -66,7 +66,7 @@ grove_gyro_values_t gyro_values;
 /*---------------------------------------------------------------------------*/
 void (*grove_gyro_int_callback)(uint8_t value);
 /*---------------------------------------------------------------------------*/
-static uint16_t
+static int16_t
```

Comment:
Good catch, but what do you think about using `int` instead as this seems to be the return type for the other functions in this file that returns `GROVE_GYRO_SUCCESS` or `GROVE_GYRO_ERROR`?

Reply:
Fixed.